.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,56 @@
+name: prepare-release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: Type of release
+        required: true
+        type: choice
+        options:
+          - PATCH
+          - MINOR
+          - MAJOR
+        default: PATCH
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v4
+
+      # SSH connection with keys is necessary to allow `git push`
+      - name: Ensure correct SSH connection
+        uses: webfactory/ssh-agent@v0.8.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
+
+      - uses: actions/setup-java@v3
+        with:
+          java-version: 11
```

Comment:
could we use java 17 here?

Reply:
I decided to use Java 11 because this is the lowest supported version we have. If we opted for taking the latest LTS, we should probably go for Java 21.